addons/website/models/assets.py

```diff
@@ -44,6 +60,60 @@ def make_scss_customization(self, url, values):
                 'copyright-gradient': 'null',
             })
 
+        google_local_fonts = values.get('google-local-fonts')
+        if google_local_fonts and google_local_fonts != 'null':
+            # "('font x': 45, 'font y': '')" -> "{'font x': 45, 'font y': ''}"
+            google_local_fonts = safe_eval.safe_eval('{%s}' % google_local_fonts[1:-1])
+            updated_font = False
+            headers_woff2 = {
+                'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.41 Safari/537.36',
+            }
```

Comment:
Maybe a comment to explain why this is necessary ? (I don't know what this is)

Reply:
Just in case: it's because we pretend to be a browser when calling Google fonts. We cannot actually request for woff2, they take the decision about the format to use for the font based on the user agent.